src/storage/v2/property_store.cpp

```diff
@@ -1023,6 +1024,18 @@ PropertyStore::~PropertyStore() {
 }
 
 PropertyValue PropertyStore::GetProperty(PropertyId property) const {
+  // if (external) {
+  //   int DUMMY_GRAPH_ELEMENT_ID = 1;
+
+  //   auto key = property.ToString();
+  //   auto properties = external_store_mock->GetDocument(DUMMY_GRAPH_ELEMENT_ID)["properties"];
+  //   if (!properties.contains(key)) {
+  //     return PropertyValue();
+  //   }
+
+  //   return PropertyValue(properties.at(key).get<int64_t>());
+  // }
```

Comment:
What is this comment, how will this work in property store, have you made any decisions?

Reply:
Oh, this comment was left over from an earlier commit. Right now we’re storing the indexed properties both in PropertyStore and in Tantivy, but the plan is to remove this redundancy. All communication with Tantivy will be done from `VertexAccessor` and not from `PropertyStore` methods.